pkg/webhook/testdata/psp-constraints/host-filesystem-constraint.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
```

Comment:
we probably want to put each test case in its own directory. If there are common files, we can split them out into a shared directory somewhere.

Reply:
+1 on having a separate directory for each test case. Change is on the way.

Reply:
Changed